src/test/groovy/graphql/nadel/StrategyTestHelper.groovy

```diff
@@ -136,6 +136,60 @@ class StrategyTestHelper extends Specification {
         return [resultData(response), resultErrors(response)]
     }
 
+
+    Object[] test1ServiceWithNHydration(GraphQLSchema overallSchema,
+                                        String serviceOneName,
+                                        GraphQLSchema underlyingOne,
+                                        String query,
+                                        List<String> topLevelFields,
+                                        List<String> expectedQueries,
+                                        List<Map> responses,
+                                        int nCalls,
+                                        ServiceExecutionHooks serviceExecutionHooks = new ServiceExecutionHooks () {},
+                                        Map variables = [:],
+                                        ResultComplexityAggregator resultComplexityAggregator
+    ) {
+        def serviceExecutionResults = []
+        for (Map response : responses) {
+            serviceExecutionResults.add(new ServiceExecutionResult(response))
+        }
+
+        boolean calledService1 = false
+        def call = -1
+        ServiceExecution service1Execution = { ServiceExecutionParameters sep ->
+            call++
+            println printAstCompact(sep.query)
```

Comment:
did you want these printlin there after you have finished?

Reply:
yeah it helps to see what the underlying query is during tests